### ISSUE TITLE
Add desktop shortcuts on Windows

### ISSUE THREAD

**Issue by InfinityGhost** [OpenTabletDriver/OpenTabletDriver#2625]
### Acknowledgements

- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.

### Description

Adding shortcuts on Windows will help a lot with the confusion of OpenTabletDriver being started in the wrong directory as we can specify the working directory in the shortcuts themselves.

**Comment by gonX** [comment]
I disagree, this should be fixed by OpenTabletDriver probing its launch path after looking in the working directory.

**Comment by gonX** [comment]
Moving to 0.6.8 milestone for further evaluation.
